internal/class.h

```diff
@@ -145,14 +145,16 @@ struct RClass {
 };
 
 // Assert that classes can be embedded in heaps[2] (which has 160B slot size)
-// TODO: restore this assertion after removing several fields from rb_classext_t
-// STATIC_ASSERT(sizeof_rb_classext_t, sizeof(struct RClass) + sizeof(rb_classext_t) <= 4 * RVALUE_SIZE);
+// On 32bit platforms there is no variable width allocation so it doesn't matter.
```

Comment:
I can't reenable that assertion if we merge: https://github.com/ruby/ruby/pull/13315